### PR TITLE
[LA64_DYNAREC] Added more opcodes

### PR DESCRIPTION
**Pull request by ksco**

### CODE REVIEW

**Comment by ksco** [comment]
Hmmm QEMU raised sigill, maybe the `AMSWAP_DB.B` instruction is too fancy? The instruction is added in Spec V1.1, it's "Atomic swap bytes with data barrier".

**Comment by ksco** [comment]
Sure, then this is ready to review once it passes.

File: src/dynarec/la64/dynarec_la64_0f.c [hunk 1]
```diff
@@ -491,6 +491,37 @@ uintptr_t dynarec64_0F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
...
+            if (MODREG) {
+                v1 = sse_get_reg(dyn, ninst, x1, (nextop & 7) + (rex.b << 3), 0);
+            } else {
+                addr = geted(dyn, addr, ninst, nextop, &wback, x2, x3, &fixedaddress, rex, NULL, 1, 1);
+                v1 = fpu_get_scratch(dyn);
+                VLD(v1, ed, fixedaddress);
+            }
+            u8 = F8;
+            q0 = fpu_get_scratch(dyn);
+            VOR_V(q0, v1, v1);
```
Comment (ptitSeb) [inline src/dynarec/la64/dynarec_la64_0f.c:521]: those 2 lines are probably useless if GX==EX.
But if GX!=EX, q0 is probably not usefull and stuff can be done in v0 directly?

File: src/dynarec/la64/dynarec_la64_0f.c [hunk 2]
```diff
@@ -491,6 +491,37 @@ uintptr_t dynarec64_0F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
...
+                addr = geted(dyn, addr, ninst, nextop, &ed, x3, x1, &fixedaddress, rex, NULL, 1, 0);
+                LD_H(gd, ed, fixedaddress);
+            }
+            if (!rex.w) ZEROUP(gd);
+            break;
+        case 0xC6:
+            INST_NAME("SHUFPS Gx, Ex, Ib");
+            nextop = F8;
+            GETGX(v0, 1);
+            if (MODREG) {
```
Comment (ptitSeb) [inline src/dynarec/la64/dynarec_la64_0f.c:512]: GETEX can probably be used there?

File: src/dynarec/la64/dynarec_la64_0f.c (hunk 1 above)
Comment (ksco) [reply src/dynarec/la64/dynarec_la64_0f.c]: Oh I missed the GX!=EX suggestion, sorry.

Comment (ksco) [reply src/dynarec/la64/dynarec_la64_0f.c]: Okay done, much better, thanks!

**Pull request merged**